relayer-cli/src/commands/query/packet.rs

```diff
@@ -83,8 +83,8 @@ impl Runnable for QueryPacketCommitmentsCmd {
             Ok(cs) => status_info!(
                 "Result for packet commitments query at height",
                 "{:?} {:#?}",
-                cs.0,
```

Comment:
FYI there will be conflicts here (due to recent changes in #518).

Reply:
Right. There is also a problem with the returned height. We cannot specify a height in the current gRPC request. So no matter what height we put in the CLI it will always return the query result from latest height. I will add this to @andynog 's PR with UX improvements.